packages/request-manager/src/torControlPort.ts

```diff
@@ -0,0 +1,111 @@
+import net, { Socket } from 'net';
+import fs from 'fs';
+import crypto from 'crypto';
+import util from 'util';
+import path from 'path';
+
+const readFile = util.promisify(fs.readFile);
+const randomBytes = util.promisify(crypto.randomBytes);
+
+const getCookieString = async (authFilePath: string) => {
+    const controlAuthCookiePath = path.join(authFilePath, 'control_auth_cookie');
+    return (await readFile(controlAuthCookiePath)).toString('hex');
+};
+
+const getClientNonce = async () => (await randomBytes(32)).toString('hex');
+
+let clientNonce: string;
```

Comment:
Is it necessary to have this `clientNonce` floating in the space instead of encapsulated in the class?

Reply:
Yes, done [20a20ea](https://github.com/trezor/trezor-suite/pull/5258/commits/20a20ea2d2fdd042c8faf715e4ff077eaa8988d4)